CONTRIBUTING.md

```diff
@@ -149,3 +149,15 @@ If you want to talk with other folks in the Pulumi community
 (including members of the Pulumi team) come hang out in the
 `#contribute` channel on the
 [Pulumi Community Slack](https://slack.pulumi.com/).
+
+## Release Process
+
+To release a new version of `pulumi-java`, create a new pull request that
+updates [CHANGELOG.md](./CHANGELOG.md) and [CHANGLOG_PENDING.md](./CHANGLOG_PENDING.md).
```

Comment:
I think we should be more explicit here since there is a multi-step process:

1. Raise a PR that copies pending to changelog. Leave pending as is -- if you empty it at this point, I think there is some GH process (e.g. generated release notes) that fails -- Justin has details IIRC.
2. Merge that PR
3. _Make sure your local `main` is up-to-date and that you do not just push your local copy of that PR_ (I have done this several times :see_no_evil: )
4. When you are sure that `main` == `origin/main`, run the release script
5. Raise another PR that empties out pending changelog when the release has completed

Reply:
Aha, it does break the release notes here https://github.com/pulumi/pulumi-java/releases :)

Reply:
(I manually updated them just now)